packages/compartment-mapper/src/import-archive.js

```diff
@@ -164,12 +165,11 @@ export const parseArchive = async (
   }
 
   const compartmentMapText = textDecoder.decode(compartmentMapBytes);
-  const compartmentMap = /** @type {CompartmentMapDescriptor} */ (parseLocatedJson(
+  const compartmentMap = parseLocatedJson(
```

Comment:
I might have asked this before, but do our specs say anything about repeated keys in JSON? That would be a way for two different readers to interpret the same compartment map in divergent ways. If TC39 doesn't have a stance, we might want to add a note to the docs around this hashing/security stuff, that correctness depends upon consistent behavior of everybody's `JSON.parse`.

Reply:
JSON (ECMA 404) has no position on repeated keys, but JS (ECMA 262) takes the position that JSON gets evaluated as a JS expression if it parses according to 404, such that the last property defined with the one key wins. So, as long as everyone uses JS, they’ll arrive at the same interpretation.